scripts/DoraGatherPDB.py

```diff
@@ -272,7 +273,7 @@ def dictify_pdb_row(pdb_row):
     # Use multiprocessor pool to retrieve various data for each PDB
     pool = Pool()
     # results = pool.map(extract_pdb_data, [(db_pdb_row.id, current_crawl_number) for db_pdb_row in db_pdb_rows])
-    results = pool.map(extract_pdb_data, db_pdb_dicts)
+    results = map(extract_pdb_data, db_pdb_dicts) #I changed pool.map() to map() to get error messages in order -MI
```

Comment:
This will not use the `multiprocessing` pool anymore.  Are you sure you want to do this?